### ISSUE TITLE
A11y_Visual Studio Code Services_Docs Java_Getting started_Focus Order : Keyboard focus is moving from Picture-in-picture at bottom to Download menu-item at top of more option button in the video.

### ISSUE THREAD

**Issue by manaswita-chougule** [microsoft/vscode-docs#6328]
### GitHub Tags
#WCAG2.4.3; #A11ySev2; #Focus Order; #A11yTCS; #A11yWCAG2.1; #Chrome; #DesktopWeb; #Win11; #Visual Studio Code Services; #BM_VisualStudioCodeServices_Web_Apr2023; #A11yAuto; #Keyboard; #TabStops;
### Environment Details
Application Name: Visual Studio Code Services
URL: [https://code.visualstudio.com/](https://code.visualstudio.com/)
Microsoft Windows 11 Enterprise 10.0.22621 Build 22621
Chrome Version 113.0.5672.93 (Official Build) (64-bit)

### Repro Steps:

1. Hit the application URL: https://code.visualstudio.com/
2. Tab till "Docs" link and press enter.
3. Tab till "Java" link in the left navigation and press enter.
4. Tab till "Getting Started" link which is present under the java and press enter.
5. Tab till Video under heading "Creating Source code file".
6. Tab till More option button.
7. Observe that Keyboard focus is moving from Picture-in-picture at bottom to Download menu-item at top.

### Actual Result
Focus is moving from bottom to top in menu-items of more option button in the video. Its moving from Picture-in-picture at bottom to download menu-item at top.
Note:
Similar issue is observed in video throughout the application.

### Expected Result
Focus should move from top to bottom and left to right i.e. from Download menu-item to Picture-in-picture of more option button in the video.

### User Impact
Keyboard and screen reader users will face difficulties in navigating within the menu-items, if it's not follows the correct focus order.

### Attachment:
![focus order -video](https://github.com/microsoft/vscode-docs/assets/91586101/4d3295dc-2fe1-4f98-9c8c-81577df747fd)

**Comment by rzhao271** [comment]
Note to self: the issue is the tab order for https://code.visualstudio.com/docs/java/java-tutorial#_creating-a-source-code-file goes to the bottommost option of the "More options" menu first. This issue might be an upstream/browser issue.

**Comment by manaswita-chougule** [comment]
@rzhao271 Could you please confirm us whether the issue is external to the chrome. Thanks!

**Comment by rzhao271** [comment]
@manaswita-chougule it is an external Chrome issue.

**Comment by manaswita-chougule** [comment]
Filed external bug with the Chrome team - https://dev.azure.com/CSS-IRT/CSS%20IRT/_workitems/edit/208165. Once the external bug is fixed, will act on the primary bug. Thanks!

**Comment by manaswita-chougule** [comment]
#Regressed:06-01-23;
@rzhao271 The issue is reproducing with both the browsers Chrome and ChromiumEdge, so it is no longer external to the Chrome team. Hence removing the external tags. Could you please look into this issue. Thanks!

**Comment by rzhao271** [comment]
@manaswita-chougule Google Chrome and Microsoft Edge are both built on top of Chromium. The issue is still an external issue.

**Comment by manaswita-chougule** [comment]
#Linked:Bug208581; #External; #External:Chromium; #Regressed:06-07-23;
Filed external bug with the Chromium team. Once the external bug is fixed, will act on the primary bug. Thanks!

**Comment by manaswita-chougule** [comment]
#PendingInput; #Regressed:06-09-23;

@rzhao271 Could you please help us with the code pen/public repro case/minimal HTML code to repro the issue. So we can share it with 3rd party chromium team. Thanks!

**Comment by rzhao271** [comment]
Hi @manaswita-chougule, the Chromium team fixed the issue!
The fix should be available in Chrome version 116.0.5841.0. To get that version, please test with [Chrome Dev](https://www.google.com/chrome/dev/). The fix is not yet available in the regular Chrome.

**Comment by manaswita-chougule** [comment]
#Regressed:07-05-23; #Closed;
Verified the issue in below environment:
URL: https://code.visualstudio.com/docs/java/java-tutorial#_creating-a-source-code-file
Chrome: Version 116.0.5845.14 (Official Build) dev (64-bit)

The issue is fixed, hence closing the bug. Please find attachment for reference. Thanks!
![focus order is correct](https://github.com/microsoft/vscode-docs/assets/91586101/8d689a8c-123f-4da5-98fc-6333adde87d6)
